src/Tranche.sol

```diff
@@ -6,10 +6,10 @@
  */
 pragma solidity ^0.8.13;
 
-import "../lib/solmate/src/auth/Owned.sol";
-import "../lib/solmate/src/mixins/ERC4626.sol";
+import {Owned} from "../lib/solmate/src/auth/Owned.sol";
+import {ERC20, ERC4626} from "../lib/solmate/src/mixins/ERC4626.sol";
```

Comment:
do we need ERC20? only used for `using SafeTransferLib for ERC20;` but I don't see any ERC20 asset transfers used